frontend/src/components/AppRoutes.jsx

```diff
@@ -11,6 +12,8 @@ function AppRoutes(){
             <Route path="/" element={<BookList/>}/>
             <Route path="/new" element={<NewBookForm/>}/>
```

Comment:
@nshelokar-vf, please follow naming conventions while naming react components.
It seems like NewBookForm is a modal rather than a component.

Reply:
Sure Sir.